routes/products.js

```diff
@@ -0,0 +1,46 @@
+// Requires
+const express = require('express');
+const jwt = require('jsonwebtoken');
+const {
+    getProductById,
+    getAllProducts,
+    createProduct,
+    destroyProduct,
+    updateProduct,
+
+} = require('../db');
+
+// Declarations
+const productsRouter = express.Router();
+
+productsRouter.get('/', async (req, res, next) => {
+    try {
+        const products = await getAllProducts()
+        if (products) {
+            res.send(products)
+        }
+        next();
```

Comment:
else next("error");

Reply:
otherwise this may result in a 404, which is not what you want to do, when handling the error

Reply:
next is only used for middleware, or handling an error